openslides/motion/templates/motion/motion_detail.html

```diff
@@ -280,6 +283,20 @@
         </h5>
         {{ version.creation_time }}
 
+        {% if 'motion_amendments_enabled'|get_config %}
+            <h5>{% trans 'Amendments' %}:</h5>
+            {% with amendments=motion.amendments.all %}
+                {% if amendments %}
+                    <ul>
+                        {% for amendment in amendments %}
+                            <li><a href="{{ amendment|absolute_url }}">{{ amendment }}</a></li>
+                        {% endfor %}
+                    </ul>
+                {% endif %}
+            {% endwith %}
+            <a class="btn btn-mini" href="{% url 'motion_create_amendment' motion.pk %}">{% trans 'New amendment' %}</a>
```

Comment:
Please use blue button with `class="btn btn-mini btn-primary"` and add the "plus"-icon as the "new motion" button.